zds/tutorialv2/publication_utils.py

```diff
@@ -148,6 +124,36 @@ def publish_content(db_object, versioned, is_major_update=True):
     return public_version
 
 
+def update_existing_publication(db_object, is_update, versioned):
+    public_version = db_object.public_version
+    is_update = True
+    # the content have been published in the past, so clean old files !
+    old_path = public_version.get_prod_path()
+    logging.getLogger(__name__).debug('erase ' + old_path)
+    shutil.rmtree(old_path)
+    # if the slug change, instead of using the same object, a new one will be created
```

Comment:
```diff
-# if the slug change, instead of using the same object, a new one will be created
+# if the slug has changed, create a new object instead of reusing the old one
``